### ISSUE TITLE
[Term Entry] Python Strings: `.endswith()`

### ISSUE THREAD

**Issue by Pandz18** [Codecademy/docs#515]
### Reason/inspiration (optional)

Hey,
I would like to create an entry for the inbuilt string library functions in python such as capitalize(). If I would like to enter more terms do I have to create multiple entries or can I piggyback on this Issue? And do I have to create different PRs for each term or just one common branch ex: "Python-string-infunct"?

### Entry Type (select all that apply)

New Concept Entry for Existing Topic

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [x] Issue is assigned

**Comment by Dusch4593** [comment]
Hey @Pandz18 there's already [an entry for Python strings](https://www.codecademy.com/resources/docs/python/strings) with a terms table of its built-in functions near the bottom. We are always happy to add more term entries for more functions, though! 😄 

> If I would like to enter more terms do I have to create multiple entries or can I piggyback on this Issue?

Going forward, we're trying to keep it to one entry per one issue. That's why we link PRs to them so that once the linked PR is merged, the issue is auto-closed. This means you would have to file a new issue each time you'd like to submit a PR for a new term entry.

> And do I have to create different PRs for each term or just one common branch ex: "Python-string-infunct"?

Please do create a different PR for each term to be added; no common branches.

**Comment by Pandz18** [comment]
Hey @Dusch4593 , 

Could I add get ".endswith()" function? 

OK thank you for clarifying that. Also another question, I have just created a PR regarding the git reset command ? If I begin to work on these new functions that I would like to add, would there be clashes if I make PRs regarding them? Or should I wait till the PR for the reset command is merged?

**Comment by Ethan-Master-Coding** [comment]
@Pandz18 You can have multiple PRs open at a time there is no reason to wait. At one time I personally had about 4 open at once.

**Comment by Dusch4593** [comment]
@Pandz18 To Ethan's point, it is totally ok to submit more than 1 PR at a time. However, I recommend no more than 2-3 so that there is plenty of opportunity to go around for all Docs contributors.

Say, for example, I wanted to create entries for 2 concepts (A and B) and one (unrelated) term. I'd do the following:

- Submit one (1) PR for concept A; only changes for that entry should be pushed to this PR.
- Submit one (1) PR for concept B; only changes for that entry should be pushed to this PR.
- Submit one (1) PR for the term entry; only changes for that entry should be pushed to this PR.

There are definitely some cases where it makes sense to attach more than one entry/file to one PR, such as one that includes a concept entry and x-amount of relevant term entries.

In conclusion: go for it! 😄 You should totally submit a new PR for the Python `.endswith()` function.
